locomotion:
  speed:
    angular_scale:  500.0           # Scale factor for angular velocity
    linear_scale:   300.0           # Scale factor for linear velocity
  motor_temp_lim:   70              # The temperature the motors will run up to, in celsius
  control_degree:   30              # How smooth the controller responds
  control_method:   "velocity"      # Velociy or torque control
  wheel_ids:        [3, 4, 2, 1]    # fr, fl, rr, rl

occupancy_map:
  resolution:       5               # Resolution of the map r, means 2**r cells per meter
  length:           50              # Length of the map in meters

general:
  loop_rate:        20              # Control loop frequency
  can_interface:    "vcan0"         # Name of the CAN network interface